src/test/scenarios/datafactory/output/src/datafactory/azext_datafactory/tests/latest/test_datafactory_scenario.py

```diff
@@ -119,7 +119,7 @@ def step__integrationruntimes_put_integrationruntimes_create(test, rg):
 # EXAMPLE: /IntegrationRuntimes/put/IntegrationRuntimes_Create
 @try_manual
 def step__integrationruntimes_put_integrationruntimes_create(test, rg):
```

Comment:
The dup definition of step__integrationruntimes_put_integrationruntimes_create is because of examples?

Reply:
should come from the generated configuration in the before version. corrected here: https://github.com/Azure/autorest.az/pull/449/commits/26b5d2447d102b3ff6723165c179581112f8b9af

Reply:
got it.